luthier/config/src/tables.lua

```diff
@@ -168,15 +167,26 @@ M.physical = {
 
 M.logical = {
     wikipedia = {
+        type = "abridged",
```

Comment:
Why `abridged`? Why not use something like `fili` since we're just building the boring old `LogicalTable` that comes
with Fili?

Reply:
It is addressed in the most recent update in the comment of tables.lua. 
Simply put, there will be another factory to handle the build with more configurable items. 

I will make that comment-amend commit live after I get all the suggestions about documentation done.

Reply:
Make this "default" rather than abridged.